ksql-engine/src/main/java/io/confluent/ksql/function/udaf/topk/TopkAggFunctionDeterminer.java

```diff
@@ -24,30 +24,57 @@
 import io.confluent.ksql.function.KsqlAggFunctionDeterminer;
 import io.confluent.ksql.function.KsqlAggregateFunction;
 import io.confluent.ksql.util.KsqlException;
+import org.apache.kafka.connect.data.SchemaBuilder;
 
 public class TopkAggFunctionDeterminer extends KsqlAggFunctionDeterminer {
+  private int topKSize = 0;
 
   public TopkAggFunctionDeterminer() {
     super("TOPK", Arrays.asList());
   }
 
+  public TopkAggFunctionDeterminer(int topKSize) {
+    this();
+    this.topKSize = topKSize;
```

Comment:
Introduce `topKSize` mainly for testing reason. To be able to specify size of the window. KSQL engine initialize this variable via `TopkKudaf#getInstance`